app/assets/stylesheets/regions/mastheads/slider-masthead.scss

```diff
@@ -1 +1,70 @@
-/* New slider masthead styles will go here */
\ No newline at end of file
+/*
+ * Slider v201611
+ */
+div.slider.version-201611 {
+  margin: 0 auto;
+  width: 1130px;
+  height: 440px;
+
+  div.grid {
+    .column {
+      float: left;
+      position: relative;
+      height: 440px;
+      overflow: hidden;
+      font-family: $font-stack-heading;
+
+      &.image {
+        width: 780px;
+
+        div.photo-by {
+          position: absolute;
+          bottom: 0;
+          left: 0;
+          padding: 16px;
+          font-size: 16px;
+          color: $white;
+          background-color: rgba(56,56,56,0.90);
+        }
+      }
+
+      &.header {
+        width: 350px;
+        color: $white;
+
+        div.aligned {
+          position: absolute;
```

Comment:
Is there a reason for the `div` selectors? and is it necessary to make the selectors this specific? I think we want to avoid nesting more than 2 or 3 levels deep if possible.